Reorder token request docs (#4463)
jupyterhub/jupyterhub: docs/source/howto/rest.md

:::

### Register API tokens via configuration

Sometimes, you'll want to pre-generate a token for access to JupyterHub,
typically for use by external services,
so that both JupyterHub and the service have access to the same value.

First, you need to generate a good random secret.
A good way of generating an API token is by running:

```bash
openssl rand -hex 32
```

> This `openssl` command generates a potential token that can then be added to JupyterHub configuration in `jupyterhub_config.py`.

Reviewer: ```suggestion
This `openssl` command generates a random token that can be added to the JupyterHub configuration in `jupyterhub_config.py`.
```

For external services, this would be registered with JupyterHub via configuration:

```python
c.JupyterHub.services = [
    {
        "name": "my-service",
        "api_token": the_secret_value,
    },
]
```

At this point, requests authenticated with the token will be associated with The service `my-service`.

```{note}
You can also load additional tokens for users via the `JupyterHub.api_tokens` configuration.

> However, this option has been softly deprecated since the introduction of services.

Reviewer: ```suggestion
However, this option has been deprecated since the introduction of services.
```
